gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/service/ConfidentialityServiceTest.java

```diff
@@ -39,6 +39,16 @@ static void beforeAll() {
         confidentialityService = new ConfidentialityService();
     }
 
+    @Test
+    void When_CreateMetaWithoutAddedSecurityIfConfidentialityCodesNotPresent_Expect_MetaWithoutSecurity() {
```

Comment:
Hmm, I think the naming of this test is unclear as to how it's different to `When_CreateMetaAndAddSecurityIfConfidentialityCodesPresent_With_ValidMetaProfileAndNoCv_Expect_MetaWithoutSecurity` ?!